ateliermecanique-ws/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM gradle:7.4 as builder
+WORKDIR /usr/src/app
+COPY src ./src
+COPY build.gradle .
+RUN ["gradle", "bootJar"]
+
+EXPOSE 8080
+
```

Comment:
We will have to figure out a way of also running our react app with docker!

Reply:
yessir